pkgs/development/python-modules/torch/default.nix

```diff
@@ -105,6 +105,14 @@ let
       rocm-runtime rocm-opencl-runtime hipify
     ];
   };
+
+  brokenConditions = {
```

Comment:
Might as well `filterAttrs (_: cond: cond)` to skip the passing tests from nix eval

Reply:
Good call; fixed in latest.